Docs/topics/discord/information.md

```diff
@@ -3,43 +3,31 @@ channelId: 1049224956162097212
 
 ## Enforcement
 
-We generally try to be hands-off when it comes to punishment. In most cases, the offending content will simply be deleted, or you will receive a verbal warning. In more extreme cases, you will be muted for anywhere between an hour and a week. This mostly happens after persisting and being told to stop doing something by a staff member.
+Generally, we try to be hands-off when it comes to punishment. Mistakes happen and everyone has off days–more often than not we might simply ask you to stop doing something and the offending content will be deleted. In more extreme cases you might receive a verbal warning from our bot–again this isn’t a punishment so to speak but something we issue in the hopes that behavior changes.
 
-More often than not, a compromise by a staff member is given. For example, one may be told to move to a different channel or thread. Not following this advice will almost definitely result in a mute.
-
-If you have proven to consistently be disrespectful to the community, only then will you be banned.
+With that being said, if you deliberately and/or repeatedly push buttons, we will not hesitate to mute or ban you depending on the severity of your offenses. Participation in our community is a privilege and we’re not obligated to keep providing service if you don’t respect that.
 
 ## Appealing
 
-In the case that you've received a punishment (such as a ban or mute), you may feel this decision has been made unfairly.
-
-We currently lack a system for proper appeals as we usually don't have enough problems to warrant one. However, we still recognize the need for one so right now our recommendation for this right now is to contact a staff member directly. However, try to avoid contacting the staff member that punished you if possible. This can lead to bias in the handling of your case, as the staff member that punished you will generally not have an opinion in your favor.
-
-Generally though, most punishments are given out for a reason. Due to our hands-off nature we tend to focus on general behaviour rather than individual incidents. For example, if your punishment was delivered as a result of bad behavior over the past month, we won't be accepting appeals regarding behavior over the past day. Some things may be what cause punishments, but more often than not they are not the whole story.
+We currently lack a proper appeals system due to lack of cases. However, if you believe a punishment was unfairly administered, you can reach out to a staff member–preferably not the one who issued the punishment (as to avoid bias)–and explain your perspective. Most punishments we issue tend to focus on general behavior rather than individual incidents so in most instances we probably won’t reverse a decision.
```

Comment:
You can also mention that I can be contacted at `support@littlebigrefresh.com` or `jvyden@jvyden.xyz` for appeals.

I think we should have a proper system for appeals by now but we can talk about that later.